Description: Avoid installing build system files into /usr/include
Author: Daniel Schepler <schepler@debian.org>
Forwarded: https://lists.launchpad.net/maria-developers/msg07924.html

--- a/libmariadb/CMakeLists.txt
+++ b/libmariadb/CMakeLists.txt
@@ -443,7 +443,10 @@ INSTALL(TARGETS
           ARCHIVE DESTINATION "${LIB_INSTALL_DIR}/${LIBSUFFIX_INSTALL_DIR}")
 
 INSTALL(DIRECTORY ${CMAKE_SOURCE_DIR}/include/
-        DESTINATION ${INCLUDE_INSTALL_DIR}/${SUFFIX_INSTALL_DIR})
+        DESTINATION ${INCLUDE_INSTALL_DIR}/${SUFFIX_INSTALL_DIR}
+        PATTERN "*.h.in" EXCLUDE
+        PATTERN "CMakeLists.txt" EXCLUDE
+        PATTERN "Makefile.am" EXCLUDE)
 INSTALL(FILES 
    ${CMAKE_BINARY_DIR}/include/my_config.h
    ${CMAKE_BINARY_DIR}/include/mysql_version.h
